runelite-client/src/main/java/net/runelite/client/plugins/antidrag/AntiDragPlugin.java

```diff
@@ -24,54 +24,82 @@
  */
 package net.runelite.client.plugins.antidrag;
 
-import com.google.inject.Provides;
-import java.awt.event.KeyEvent;
 import javax.inject.Inject;
-import net.runelite.api.Client;
+import java.awt.event.KeyEvent;
+import com.google.inject.Provides;
+import net.runelite.api.GameState;
+import net.runelite.api.Varbits;
 import net.runelite.api.events.FocusChanged;
+import net.runelite.api.events.WidgetLoaded;
 import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetID;
 import net.runelite.api.widgets.WidgetInfo;
-import net.runelite.client.config.ConfigManager;
+import net.runelite.client.callback.ClientThread;
 import net.runelite.client.eventbus.Subscribe;
+import net.runelite.api.Client;
+import net.runelite.api.events.VarbitChanged;
+import net.runelite.client.config.ConfigManager;
+import net.runelite.client.events.ConfigChanged;
 import net.runelite.client.input.KeyListener;
 import net.runelite.client.input.KeyManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
 
+
 @PluginDescriptor(
-	name = "Shift Anti Drag",
+	name = "Anti Drag",
+	enabledByDefault = false,
 	description = "Prevent dragging an item for a specified delay",
 	tags = {"antidrag", "delay", "inventory", "items"}
 )
 public class AntiDragPlugin extends Plugin implements KeyListener
 {
-	private static final int DEFAULT_DELAY = 5;
+	static final String CONFIG_GROUP = "antiDrag";
+
+	private final int DEFAULT_DELAY = 5;
 
 	@Inject
 	private Client client;
 
+	@Inject
+	private ClientThread clientThread;
+
 	@Inject
 	private AntiDragConfig config;
 
 	@Inject
 	private KeyManager keyManager;
 
+	private boolean lastStatus; //true = was in PvP, false = was not in Pvp
```

Comment:
`lastInPvp` would be a much better var name, which would completely remove the need for this added comment.